anchor_gpt/prompt_store.py

```diff
@@ -1,72 +1,106 @@
 import sqlite3
 import uuid
-import json
+import ast
 
 sqlite3.register_adapter(uuid.UUID, lambda u: u.bytes_le)
 
 from .prompt import Prompt
 
 class PromptStore:
+    """
+    An extensible class for prompt stores.
+    """
     def add(self, prompt):
-        '''
+        """
         Add a prompt to the store.
-        prompt: a Prompt object.
-        '''
+
+        Parameters:
+            prompt: a Prompt object.
+        """
         raise NotImplementedError
-    
+
     def update(self, prompt):
-        '''
+        """
         Update a prompt in the store.
-        prompt: a Prompt object.
-        '''
+
+        Parameters:
+            prompt: a Prompt object.
+        """
         raise NotImplementedError
-    
+
     def get_by_ids(self, ids):
-        '''
+        """
         Get a list of prompts by their ids.
-        ids: a list of UUIDs.
-        '''
+
+        Parameters:
+            ids: a list of UUIDs.
+        """
         raise NotImplementedError
-    
+
+    def purge(self):
+        """
+        Purge the store of all prompts.
+
+        """
+        raise NotImplementedError
+
 
 def prompt_to_db(prompt):
-    return (prompt.id, prompt.text, str(prompt.scores) if prompt.scores else None, str(prompt.embeddings) if prompt.embeddings else None)
+    """
```

Comment:
this is an internal util, i don't think a docstring is needed

Reply:
ok, but it can't hurt